frontend/src/components/Navigation/Desktop/Buttons/NewActionButton/NewActionButton.tsx

```diff
@@ -0,0 +1,67 @@
+import AddIcon from '@mui/icons-material/Add';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+import { Button, ListItemIcon, ListItemText, Menu, MenuItem, useTheme } from '@mui/material';
+import { useState } from 'react';
+
+interface IMenuItem {
+  label: string;
+  href: string;
+}
+
+/**
+ * A simple button that opens a menu for a new action.
+ * @returns
+ */
+function NewActionButton(): JSX.Element {
```

Comment:
Move all buttons' files up in the `buttons` directory.
What we currently have is:
- `/Buttons/NewActionButton/NewActionButton.tsx`
- `/Buttons/SearchButton/SearchButton.tsx`
- ...

Move to:
- `/buttons/NewActionButton.tsx`
- `/buttons/SearchButton.tsx`
- ...

Creating subdirectories for each file is unnecessary and will likely bring complications.